platform/Cmd.roc

```diff
@@ -24,7 +25,14 @@ interface Cmd
 Cmd := InternalCommand.Command implements [Inspect]
 
 ## Errors from executing a command.
-Error : InternalCommand.CommandErr
+Err : InternalCommand.CommandErr
+
+outputErrToStr : (Output, Err) -> Str
+outputErrToStr = \err ->
+    when err is
+        (_, (ExitCode code)) -> "Child exited with non-zero code: $(Num.toStr code)"
+        (_, (KilledBySignal)) -> "Child was killed by signal"
```

Comment:
Heads up: `(KilledBySignal)` is not really a tuple (just a parenthesized expr), if that's what you had intended. Seems kinda misleading to parenthesize it.

Reply:
Thanks, updated